packages/grpc-js/src/server.ts

```diff
@@ -223,7 +224,25 @@ export class Server {
   }
 
   forceShutdown(): void {
-    throw new Error('Not yet implemented');
+    if (this.http2Server === null) {
+      throw new Error('server is not running');
```

Comment:
This should not be treated as an error. Just make it a no-op if there's no server. The same is true for `tryShutdown`.